src/Infrastructure/Rokalo.Infrastructure.Db.Users/Configurations/UserEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Rokalo.Infrastructure.Db.Users.Configurations
+{
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.EntityFrameworkCore.Metadata.Builders;
+    using Rokalo.Domain;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    internal sealed class UserEntityTypeConfiguration : IEntityTypeConfiguration<User>
+    {
+        public void Configure(EntityTypeBuilder<User> builder)
+        {
+            builder.ToTable("Users");
+
+            builder.HasKey(key => key.Id);
+
+            builder.Property(p => p.Email).IsRequired();
+
+            builder.Property(p => p.Password).IsRequired();
+
+            builder.Property(p => p.IsEmailVerified).IsRequired();
+
+            builder.Property(p => p.EmailVerificationCode).IsRequired();
+
+            builder.HasOne(x => x.Profile);
```

Comment:
I think that you can define foreing key here. Take a look at the docs: https://learn.microsoft.com/en-us/ef/core/modeling/relationships.

Idea is that we want to access everying when we get user. For example, when we get user, we want to see his profile and his claims. We don't want it other way around (although it is possible by adding User in ProfileEntityTypeConfiguration - at the moment, we don't want that).